### PR TITLE
Fixed #29836 -- Bumped required cx_Oracle to 6.0.

### PR DESCRIPTION
**Pull request by felixxm**
Ticket [29836](https://code.djangoproject.com/ticket/29836).

### CODE REVIEW

File: tests/requirements/oracle.txt [hunk 1]
```diff
@@ -1 +1 @@
-cx_oracle
+cx_oracle >= 6.0
```
Comment (felixxm) [inline tests/requirements/oracle.txt]: I'm not sure if this is required :thinking: .

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: docs/releases/2.2.txt [hunk 2]
```diff
@@ -140,6 +140,8 @@ Database backends
 
 * Added result streaming for :meth:`.QuerySet.iterator` on SQLite.
 
+* Support for ``cx_Oracle`` < 6.0 is removed.
```
Comment (timgraham) [inline docs/releases/2.2.txt]: Put under "Backwards incompatible" not "Minor Features" :-)

Comment (felixxm) [reply docs/releases/2.2.txt]: :man_facepalming:

**Comment by felixxm** [comment]
@timgraham Thanks for the review.

**Pull request merged**